test/config/utility.cc

```diff
@@ -1177,6 +1178,31 @@ bool ConfigHelper::setListenerAccessLog(const std::string& filename, absl::strin
   return true;
 }
 
+void ConfigHelper::initializeTlsKeyLog(
+    envoy::extensions::transport_sockets::tls::v3::CommonTlsContext& tls_context, int mode) {
```

Comment:
Please use some names flags for mode, or just use two separate booleans.

Reply:
will fix